uint8 axis
---
uint32 motor_error